config/locales/en.yml

```diff
@@ -1333,6 +1333,8 @@ en:
         help: "Notes can be created for students, groups, or assignments. They are visible to admins and graders, but not students."
 
     criteria_edit_total_mark:     "This criterion will be marked out of %{criterion_total_mark}"
+    criterion_created_success:   "Criterion was successfully created"
+    criterion_created_failture:   "Criterion failed to be created"
```

Comment:
This is spelled wrong. Also, you should add these two keys for the other locale files (you can use Google translate if you like).